source/includes/harvest/_candidates.md

```diff
@@ -1271,7 +1280,14 @@ curl -X POST 'https://harvest.greenhouse.io/v1/candidates'
             }
         }
     ],
-    "educations": [],
+    "educations": [
+        {
+            "id": 459,
```

Comment:
@tdphillipsjr Looking at this again, is it confusing that we use School ID as 459 in the example, then use 459 again in the response as the education ID? Would it be clearer to use different numbers in this case? Or is that only confusing to PMs and not engineers?

Reply:
I also found one typo this morning while you're in there: 

![screen shot 2018-01-19 at 11 34 55 am](https://user-images.githubusercontent.com/18620218/35160997-dd68d154-fd0c-11e7-8d56-fab878738c8a.png)
 
Should be "/degrees"